openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/course_outline.html

```diff
@@ -0,0 +1,17 @@
+<div id='course_title' class='h2'>{{ course_outline.display_name }}</div>
+<hr>
+<div id='outline' class='list-group'>
+{% for chapter in course_outline.children %}
+  {% include 'openedx_lti_tool_plugin/course_outline_item.html' with block=chapter %}
+  <div id='{{ chapter.type }}-{{ chapter.id|urlencode }}' class='list-group collapse show'>
```

Comment:
```suggestion
  <div id='{{ chapter.type }}-{{ forloop.counter }}' class='list-group collapse show'>
```

What do you think of using the forloop counter for the ids?
https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#for